<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>40</y>
      <width>141</width>
      <height>116</height>
     </rect>
    </property>
    <property name="title">
     <string>选择你喜欢的音乐风格</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QCheckBox" name="folkSongCheckBox">
       <property name="text">
        <string>民谣</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_6">
       <property name="text">
        <string>流行</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_7">
       <property name="text">
        <string>摇滚</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_8">
       <property name="text">
        <string>DJ</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>40</y>
      <width>111</width>
      <height>172</height>
     </rect>
    </property>
    <property name="title">
     <string>筛选壁纸</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QCheckBox" name="wallPaperLabel">
       <property name="text">
        <string>标签</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="widget" native="true">
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="leftMargin">
         <number>15</number>
        </property>
        <item>
         <widget class="QCheckBox" name="wallPaperAbstract">
          <property name="text">
           <string>抽象</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="wallPaperAnimal">
          <property name="text">
           <string>动物</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="wallPaperAnimation">
          <property name="text">
           <string>动漫</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="wallPaperMiddleAges">
          <property name="text">
           <string>中世纪</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="wallPaperInterCelebrity">
          <property name="text">
           <string>网红事物</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
